idaes/generic_models/unit_models/plate_heat_exchanger.py

```diff
@@ -284,63 +295,108 @@ def _make_params(self):
-                self.config.divider_plate_number
-            _plate_pitch = self.config.plate_pact_length*pyunits.m / _total_plate_number
+                self.divider_plate_number
+            _plate_pitch = self.plate_pact_length / _total_plate_number
 
-            _plate_gap = _plate_pitch - self.config.plate_thickness*pyunits.m
+            _plate_gap = _plate_pitch - self.plate_thickness
+            self.plate_gap = Expression(expr=_plate_gap)
         else:
-            _plate_gap = self.config.plate_gap*pyunits.m
-
-        self.plate_gap = Expression(expr=_plate_gap)
+            self.plate_gap = Param(mutable=True,
+                                   units=pyunits.m,
+                                   doc="Gap bewteen two plates ")
+            set_param_value(self, "plate_gap", pyunits.m)
 
         # Surface enlargement factor
+        # projected plate area = plate length *plate width
         if self.config.surface_enlargement_factor is None:
-            _projected_plate_area = (
-                _effective_plate_length * _effective_plate_width)
-            _surface_enlargement_factor = (
-                self.plate_area / _projected_plate_area)
+            self.surface_enlargement_factor = Expression(
+                expr=self.plate_area / (self.plate_length * self.plate_width))
         else:
-            _surface_enlargement_factor = \
-                self.config.surface_enlargement_factor
-
-        self.surface_enlargement_factor = Expression(
-            expr=_surface_enlargement_factor)
+            self.surface_enlargement_factor = Param(mutable=True,
+                                                    units=pyunits.dimensionless,
+                                                    doc="Surface enlargement factor ")
+            set_param_value(self, "surface_enlargement_factor",
+                            pyunits.dimensionless)
 
         # Channel equivalent diameter
-        self.channel_dia = Expression(expr=2 * self.plate_gap /
-                                      _surface_enlargement_factor,
-                                      doc=" Channel equivalent diameter")
+        self.channel_diameter = Expression(expr=2 * self.plate_gap /
+                                           self.surface_enlargement_factor,
+                                           doc=" Channel equivalent diameter")
 
         # heat transfer parameters
-        self.param_a = Var(initialize=0.3, bounds=(0.2, 0.4), units=None,
-                           doc='Nusselt parameter')
-        self.param_b = Var(initialize=0.663, bounds=(0.3, 0.7), units=None,
-                           doc='Nusselt parameter')
-        self.param_c = Var(initialize=1 / 3.0, bounds=(1e-5, 2), units=None,
-                           doc='Nusselt parameter')
-        self.param_a.fix(0.4)
-        self.param_b.fix(0.663)
-        self.param_c.fix(0.333)
+        self.nusselt_param_a = Var(initialize=0.3,
```

Comment:
Are these really `Vars`? These look like coefficients for a correlation, and thus I question whether these should be varied during optimization of the unit.